pkgs/development/python-modules/pytorch/default.nix

```diff
@@ -159,9 +159,9 @@ in buildPythonPackage rec {
 
   USE_MKL = blas.implementation == "mkl";
 
-  # Unlike MKL, MKLDNN is FOSS, so we enable support for it by default. Note
-  # that this was renamed to dnnl and then renamed again to oneDNN upstream, but
-  # pytorch still calls it by the old name mkldnn.
+  # Unlike MKL, oneDNN (née MKLDNN) is FOSS, so we enable support for
+  # it by default. PyTorch currently uses its own vendored version
+  # of oneDNN through Intel iDeep.
```

Comment:
Sigh vendored dependencies . . . I guess it is what it is though.

Reply:
Yeah, it's a bit annoying that they do not provide some option to provide your own oneDNN. Doesn't help that CMake is hidden behind their own script that gates build flags :/.

Reply:
It would be really nice if you could contribute a patch upstream to support non-vendored distribution deps, but in the meantime this PR seems OK as it at least faithfully represents the status quo.

Reply:
I will do that if I find some time. First some more BLAS impl fights, erm. I mean benchmarks. Intel has removed `MKL_DEBUG_CPU_TYPE` since MKL 2020 release 1, so Ryzen performance is abysmal. There is a workaround, but not too happy with it ;).